File: PetShop.cpp
#include <iostream>
#include <string>

using namespace std;

class PetShop{
    private:
        string ID;
        string nama;
        string kategori;
        int harga;
    public:
        PetShop(){
            ID = "";
            nama = "";
            kategori = "";
            harga = 0;
        }
        string get_ID(){
            return ID;
        }
        string get_nama(){
            return nama;
        }
        string get_kategori(){
            return kategori;
        }
        int get_harga(){
            return harga;
        }

        void set_ID(string ID_baru){
            ID = ID_baru;
        }
        void set_nama(string nama_baru){
            nama = nama_baru;
        }
        void set_kategori(string kategori_baru){
            kategori = kategori_baru;
        }
        void set_harga(int harga_baru){
            harga = harga_baru;
        }
        ~PetShop(){}
};

File: PHP/PetShop.php
<?php

class PetShop{
    private $ID = "";
    private $name = "";
    private $kategori = "";
    private $harga = "";
    private $foto = "";
    function __construct($ID, $name, $kategori, $harga, $foto){
        $this->ID  = $ID;
        $this->name = $name;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->foto = $foto;
    }

    // Setter dan Getter untuk atribut yang ada
    function setID($ID){
        $this->ID = $ID;
    }
    function getID(){
        return $this->ID;
    }

    function setName($name){
        $this->name = $name;
    }
    function getName(){
        return $this->name;
    }
    
    function setKategori($kategori){
        $this->kategori = $kategori;
    }
    function getKategori(){
        return $this->kategori;
    }

    function setHarga($harga){
        $this->harga = $harga;
    }
    function getHarga(){
        return $this->harga;
    }

    function setFoto($foto){
        $this->foto = $foto;
    }
    function getFoto(){
        return $this->foto;
    }


    // Method untuk menampilkan data yang ada
    function display(){
        echo "ID        : " . $this->ID . "\n";
        echo "Name      : " . $this->name . "\n";
        echo "Kategori  : " . $this->kategori . "\n";
        echo "Harga     : " . $this->harga . "\n";
    }
}

?>